### PR TITLE
Register OidcIdentityStoreBean for issue #21169

### PR DESCRIPTION
**Pull request by teddyjtorres**
For issue #21169

### CODE REVIEW

**Comment by teddyjtorres** [comment]
#build

File: dev/io.openliberty.security.jakartasec.3.0.internal.cdi/test/io/openliberty/security/jakartasec/cdi/extensions/OidcIdentityStoreBeanTest.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+        Set<Type> types = oidcIdentityStoreBean.getTypes();
+        String expectedTypeString = new TypeLiteral<IdentityStore>() {
+        }.getType().toString();
+    
+        assertEquals("Must be a TypeLiteral<IdentityStore> type.", expectedTypeString, types.iterator().next().toString());
+    }
+
+    @Test
+    public void testGetBeanClass() {
+        assertEquals("Must be ApplicationScoped.", IdentityStore.class, oidcIdentityStoreBean.getBeanClass());
```
Comment (ayoho) [inline dev/io.openliberty.security.jakartasec.3.0.internal.cdi/test/io/openliberty/security/jakartasec/cdi/extensions/OidcIdentityStoreBeanTest.java:89]:
```suggestion
        assertEquals("Must be an IdentityStore class.", IdentityStore.class, oidcIdentityStoreBean.getBeanClass());
```
Copy-paste error.

**Review by ayoho — APPROVED** [review]

File: dev/io.openliberty.security.jakartasec.3.0.internal.cdi/test/io/openliberty/security/jakartasec/cdi/extensions/OidcIdentityStoreBeanTest.java (hunk 1 above)
Comment (teddyjtorres) [reply dev/io.openliberty.security.jakartasec.3.0.internal.cdi/test/io/openliberty/security/jakartasec/cdi/extensions/OidcIdentityStoreBeanTest.java:89]: I agree : ) . I'll fix it in the next PR. Thank you!

**Comment by teddyjtorres** [comment]
The personal build failure is not related to the code being delivered.

**Pull request merged**